packages/glibc.rb

```diff
@@ -3,169 +3,259 @@
     })
-    binary_sha256 ({
-      i686: '52145b65cb49c2751f69d4c46636f0685f2abb6685d8a080b71b2f091595a950',
+    binary_sha256({
+      i686: '52145b65cb49c2751f69d4c46636f0685f2abb6685d8a080b71b2f091595a950'
     })
   when '2.27'
     source_url 'https://ftpmirror.gnu.org/glibc/glibc-2.27.tar.xz'
     source_sha256 '5172de54318ec0b7f2735e5a91d908afe1c9ca291fec16b5374d9faadfc1fc72'
-    binary_url ({
+
+    binary_url({
       aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/glibc/2.27_armv7l/glibc-2.27-chromeos-armv7l.tar.xz',
        armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/glibc/2.27_armv7l/glibc-2.27-chromeos-armv7l.tar.xz',
-       x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/glibc/2.27_x86_64/glibc-2.27-chromeos-x86_64.tar.xz',
+       x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/glibc/2.27_x86_64/glibc-2.27-chromeos-x86_64.tar.xz'
     })
-    binary_sha256 ({
+    binary_sha256({
       aarch64: '64b4b73e2096998fd1a0a0e7d18472ef977aebb2f1cad83d99c77e164cb6a1d6',
        armv7l: '64b4b73e2096998fd1a0a0e7d18472ef977aebb2f1cad83d99c77e164cb6a1d6',
-       x86_64: '5fe94642dbbf900d22b715021c73ac1a601b81517f0da1e7413f0af8fbea7997',
+       x86_64: '5fe94642dbbf900d22b715021c73ac1a601b81517f0da1e7413f0af8fbea7997'
+    })
+  when '2.32' # All architectures with updates past M92.
+    source_url 'https://ftpmirror.gnu.org/glibc/glibc-2.32.tar.xz'
+    source_sha256 '1627ea54f5a1a8467032563393e0901077626dc66f37f10ee6363bb722222836'
+
+    binary_url({
+      x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/glibc/2.32_x86_64/glibc-2.32-chromeos-x86_64.tpxz'
+    })
+    binary_sha256({
+      x86_64: '0e6749f4d4c445e27bcd8669865fa060193552ecda2233f7f0d5e91c90b91a0b'
     })
   end
 
   depends_on 'gawk' => :build
+  depends_on 'hashpipe' => :build
   depends_on 'libgd' => :build
+  depends_on 'libidn2' => :build
   depends_on 'texinfo' => :build
 
   def self.patch
-    # Apply patch due to new version of binutils which causes compilation failure
-    # http://lists.busybox.net/pipermail/buildroot/2017-August/199812.html
-    Dir.chdir "misc" do
-      if File.readlines("regexp.c").grep(/monitor/).any?
-        puts "Patched!".lightgreen
-      else
-        system "sed -i 's,char \\*loc1,char \\*loc1 __attribute__ ((nocommon)),g' regexp.c"
-        system "sed -i 's,char \\*loc2,char \\*loc2 __attribute__ ((nocommon)),g' regexp.c"
-        system "sed -i 's,char \\*locs,char \\*locs __attribute__ ((nocommon)),g' regexp.c"
+    case LIBC_VERSION
+    when '2.23', '2.27'
+      # Apply patch due to new version of binutils which causes compilation failure
+      # http://lists.busybox.net/pipermail/buildroot/2017-August/199812.html
+      Dir.chdir 'misc' do
+        unless File.readlines('regexp.c').grep(/monitor/).any?
+          system "sed -i 's,char \\*loc1,char \\*loc1 __attribute__ ((nocommon)),g' regexp.c"
```

Comment:
Use regular expression to merge `sed`s ?
```
sed -I 's,char \\*loc(1|2|s),char \\*loc(1|2|s) __attribute__ ((nocommon)),g' regexp.c

Reply:
@uberhacker Does that look good to you?

Reply:
Looks fine.